client/my-sites/comments/comment/style.scss

```diff
@@ -424,6 +424,7 @@
 
 	.form-textarea {
 		margin: 0px 8px 20px 8px;
+		resize: vertical;
```

Comment:
This doesn't seem to be supported in IE: https://www.w3schools.com/cssref/css3_pr_resize.asp

Reply:
FYI, we already use it for the reply textarea.
Though, I'd say that this is good to go anyway since AFAIK there is no easy workaround for `resize` on IE, and it's not really a bug, more of a polish thing.